src/main/resources/application.properties

```diff
@@ -1,17 +1,18 @@
 #Server properties
-server.port=9004
+server.port=8080
 #Datasource properties
-spring.datasource.driverClassName=com.mysql.jdbc.Driver
-spring.datasource.url=jdbc:mysql://localhost:3306/login_db
-spring.datasource.username=root
-spring.datasource.password=pass
+spring.datasource.driverClassName=com.mysql.cj.jdbc.Driver
+spring.datasource.url=jdbc:mysql://localhost:3306/login_db?createDatabaseIfNotExist=true&useUnicode=true&useJDBCCompliantTimezoneShift=true&useLegacyDatetimeCode=false&serverTimezone=Asia/Jakarta&useSSL=false&allowPublicKeyRetrieval=true
+spring.datasource.username=pass
+spring.datasource.password=root
```

Comment:
Please do not change spring jpa/datasouce default properties

Reply:
They can change manually based on their env right?

It won't effect the code